### DISCUSSION TITLE
Extended day option

### DISCUSSION THREAD

**Discussion by BeeRando** [Zrips/CMI#2271] [Feature Requests]
**Description of feature/improvement request:**
An option int he config to change the length of day/night/dusk/dawn.
---
### Detailed explanation of what it should do, when and in what situation: ###
Can change the length of it. Like make it 2 or three times longer.
### Basic explanation as to why this feature is needed and/or useful to have: ###
It'd be a cool, basic feature that many servers would really enjoy.

**Comment by sandwally** [comment]
Already possible.
Search for "TimeSpeed" in the config
